docs/core/howto/defer-intro.rst

```diff
@@ -401,8 +401,10 @@ Coroutines with async/await
 
     .. versionadded:: 16.4
 
-On Python 3.5 and higher, the :pep:`492` ("Coroutines with async and await syntax") "await" functionality can be used with Deferreds by the use of :api:`twisted.internet.defer.ensureDeferred <ensureDeferred>`.
-It is similar to ``inlineCallbacks``, except that it uses the ``await`` keyword instead of ``yield``, the ``return`` keyword instead of ``returnValue``, and is a function rather than a decorator.
+On Python 3.5 and higher, the :pep:`492` ("Coroutines with async and await syntax") "await" functionality can be used with Deferreds by the use of :api:`twisted.internet.defer.coroFnToDeferredFn <coroFnToDeferredFn>`.
+
+It is similar to ``inlineCallbacks``, except that it uses the ``await`` keyword instead of ``yield``, the ``return`` keyword instead of ``returnValue``.
+"await" functionality can also be used with Deferreds by the use of :api:`twisted.internet.defer.ensureDeferred <ensureDeferred>`. It is similar to ``coroFnToDeferredFn``, except that it is a function rather than a decorator.
```

Comment:
I think the decorator is important enough to come with an example, rather than being listed only as an aside.